AST panic: MemberExpr

```
panic: could not match regexp with string
^(?P<address>[0-9a-fx]+) <(?P<position>.*)> '(?P<type>.*?)'(?P<type2>:'.*?')? lvalue(?P<bitfield> bitfield)? (?P<pointer>[->.]+)(?P<name>\w+)? (?P<address2>[0-9a-fx]+)[\s]*$
0x3180ef0 <col:32, col:48> 'int' .n 0x317f450


goroutine 8 [running]:
github.com/elliotchance/c2go/ast.groupsFromRegex(0xc4205328f0, 0xad, 0xc420195646, 0x2d, 0x200000000040f14f)
        /home/konstantin/go/src/github.com/elliotchance/c2go/ast/ast.go:250 +0x3cc
github.com/elliotchance/c2go/ast.parseMemberExpr(0xc420195646, 0x2d, 0x5e61d9)
        /home/konstantin/go/src/github.com/elliotchance/c2go/ast/member_expr.go:27 +0x62
github.com/elliotchance/c2go/ast.Parse(0xc42019563b, 0x38, 0x5e4efa, 0x5)
        /home/konstantin/go/src/github.com/elliotchance/c2go/ast/ast.go:158 +0x125d
main.convertLinesToNodes(0xc42019e6f0, 0x225, 0x225, 0x0, 0x0, 0x0)
        /home/konstantin/go/src/github.com/elliotchance/c2go/main.go:96 +0x196
main.convertLinesToNodesParallel.func1.2(0xc420056d20, 0xc42000c7a0, 0xc42019e6f0, 0x225, 0x225, 0x0)
        /home/konstantin/go/src/github.com/elliotchance/c2go/main.go:128 +0x59
created by main.convertLinesToNodesParallel.func1
        /home/konstantin/go/src/github.com/elliotchance/c2go/main.go:132 +0x1b6

```

--- a/ast/member_expr.go
+++ b/ast/member_expr.go
@@ -19,7 +19,7 @@ func parseMemberExpr(line string) *MemberExpr {
 		`<(?P<position>.*)>
 		 '(?P<type>.*?)'
 		(?P<type2>:'.*?')?
-		 lvalue
+		(?P<lvalue> lvalue)?
 		(?P<bitfield> bitfield)?
 		 (?P<pointer>[->.]+)
 		(?P<name>\w+)?
@@ -39,7 +39,7 @@ func parseMemberExpr(line string) *MemberExpr {
 		Type2:      type2,
 		IsPointer:  groups["pointer"] == "->",
 		Name:       groups["name"],
-		IsLvalue:   true,
+		IsLvalue:   len(groups["lvalue"]) > 0,
 		IsBitfield: len(groups["bitfield"]) > 0,
 		Address2:   groups["address2"],
 		ChildNodes: []Node{},

--- a/ast/member_expr_test.go
+++ b/ast/member_expr_test.go
@@ -78,6 +78,18 @@ func TestMemberExpr(t *testing.T) {
 			IsPointer:  false,
 			ChildNodes: []Node{},
 		},
+		`0x3180ef0 <col:32, col:48> 'int' .n 0x317f450`: &MemberExpr{
+			Addr:       0x3180ef0,
+			Pos:        NewPositionFromString("col:32, col:48"),
+			Type:       "int",
+			Type2:      "",
+			IsLvalue:   false,
+			IsBitfield: false,
+			Name:       "n",
+			Address2:   "0x317f450",
+			IsPointer:  false,
+			ChildNodes: []Node{},
+		},
 	}
 
 	runNodeTests(t, nodes)

